src/lib/DiscordWorker.ts

```diff
@@ -99,5 +100,7 @@ export class DiscordWorker<T extends DiscordJob> extends Worker<any> {
       concurrency: 10,
       ...options,
     })
+
+    this.queue = new Queue(name, { connection: redis })
```

Comment:
Since we have multiple instance of each worker, this would also create multiple queues with the same name. I was wondering if this would be a problem, but it seems like Bull MQ handles this.

From their documentation: https://docs.bullmq.io/guide/queues
> When you instantiate a Queue, BullMQ will just upsert a small "meta-key", so if the queue existed before it will just pick it up and you can continue adding jobs to it.

Reply:
No the worker object is global the job instance is many. It should be fine to have the queue on the worker.